infra/src/main/java/com/redhat/developer/infra/utils/CloudEventUtils.java

```diff
@@ -0,0 +1,69 @@
+package com.redhat.developer.infra.utils;
+
+import java.net.URI;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.developer.infra.utils.exceptions.CloudEventDeserializationException;
+import com.redhat.developer.infra.utils.exceptions.CloudEventSerializationException;
+
+import io.cloudevents.CloudEvent;
+import io.cloudevents.core.builder.CloudEventBuilder;
+import io.cloudevents.jackson.JsonCloudEventData;
+import io.cloudevents.jackson.JsonFormat;
+
+public class CloudEventUtils {
```

Comment:
Any reason not to make this an `@ApplicationScope` bean? That way we could inject the `ObjectMapper` into this class and then inject the `CloudEventUtils` where it is needed. Removes the need for `static` methods and gives us more control over our usage of `ObjectMapper` throughout the code.

Reply:
The idea is the following: this util class is inside the `infra` package so that it can be used by every module that needs to encode/decode a cloud event (for example `ingress` and `executor`). 
If the module that imports it does not register the  `cloudEventJacksonModule` in the ObjectMapper, this class will not work properly: this is why I would keep this class in this package with its own ObjectMapper. 
The other modules can register or not the `cloudEventJacksonModule` without impacting this utility class. 
For example, the ingress do register it so that we can get the CloudEvent obj directly in the rest endpoint and let jackson deserialize it like magic. But, probably the `executor` do not need rest api and will not work with the serialization of the cloud event outside this class: it will probably not register the  `cloudEventJacksonModule` in its object mapper. 

In conclusion, for the current design I think we should keep this class as it is, self-contained. wdyt?

Reply:
Ah OK. That makes sense to me. Thanks.